Build/MQTTnet.nuspec

```diff
@@ -2,7 +2,7 @@
 <package >
   <metadata>
     <id>MQTTnet</id>
-    <version>0.0.0</version>
+    <version>3.0.10</version>
```

Comment:
Why did you added the versions here? They are set by the build script automatically.

Reply:
.... same reason as above